### DISCUSSION TITLE
How to define testPlan within gradle script

### DISCUSSION THREAD

**Discussion by nagkumar** [qualersoft/jmeter-gradle-plugin#175] [Q&A]
Can any share ways to define testPlan with load parameters in gradle script using this plugin..

**Comment by mathze** [comment]
I'm not quite sure if I understand you correctly. From what I've understand, you want to pass values from gradle-script which than are available in your test plan, e.g. your test plan has a property 'numberOfConcurrentUsers' and you configure it's value through the plugin, right?
If so, it should be possible through the `jmeterProperties` property and than using jmeters `__P()` function in your testplan.
Maybe [this](https://octoperf.com/blog/2019/01/14/flexible-test-plans/#command-line-properties) post may give you further ideas.
